### DISCUSSION TITLE
id.theasianparent.com

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1461] [General]
https://id.theasianparent.com/tips-merangsang-payudara?
ht![20230720_202057_rmscr.jpg](https://github.com/realodix/AdBlockID/assets/114223791/b0a1440a-8b66-4728-a1db-08d681656d30)

Notifikasi gk bisa di hilangkan
Di klik konfirmasi gk respon

**Comment by realodix** [comment]
Di Saya gak ada masalah, pas diklik akan bukan tab baru untuk login dengan akun Google

**Comment by IoBlight** [comment]
> Di klik konfirmasi gk respon

Mungkin ada domain google yang keblock kali Gan? accounts.google.com misalnya. Di saya bisa firefox mobile pakai ublock default filters
